src/server/req_rescq.c

```diff
@@ -242,7 +242,7 @@ cnvrt_timer_init()
  * @retval	non-zero	: error code if problem occurs
  */
 int
-assign_resv_resc(resc_resv *presv, char *vnodes)
+assign_resv_resc(resc_resv *presv, char *vnodes, int svr_init)
```

Comment:
Update the function header with the new parameter svr_init along with its purpose.